app.py

```diff
@@ -18,10 +18,12 @@
 LANGS = dict(zip(constants.get('GH_REPOS').split(','), constants.get('LANGS').split(';')))
 CURRENT = dict(zip(constants.get('GH_REPOS').split(','), constants.get('CURRENT').split(';')))
 
-try:
-  s3 = S3(constants.get('AWS_ACCESS_KEY'), constants.get('AWS_SECRET_KEY'), constants.get('AWS_BUCKET'))
-except:
-  s3 = None
+s3 = None
+if constants.get('GH_COMMENT').lower() == 'true':
```

Comment:
You don't actually need any of the changes in this file, because connecting to S3 will fail if the constants are not set. Then later on we can keep the old `if not s3`. This de-couples S3 usage from commenting, in case we want it for something else later.